src/pokemon_data.c

```diff
@@ -15907,3 +15905,603 @@ const u16 gChimechoLevelUpLearnset[] = {
+    }}, {{ // Numel
+        {EVO_LEVEL, 33, SPECIES_CAMERUPT},
+    }}, {{ // Camerupt
+    }}, {{ // Spheal
+        {EVO_LEVEL, 32, SPECIES_SEALEO},
+    }}, {{ // Sealeo
+        {EVO_LEVEL, 44, SPECIES_WALREIN},
+    }}, {{ // Walrein
+    }}, {{ // Cacnea
+        {EVO_LEVEL, 32, SPECIES_CACTURNE},
+    }}, {{ // Cacturne
+    }}, {{ // Snorunt
+        {EVO_LEVEL, 42, SPECIES_GLALIE},
+    }}, {{ // Glalie
+    }}, {{ // Lunatone
+    }}, {{ // Solrock
+    }}, {{ // Azurill
+        {EVO_FRIENDSHIP, 0, SPECIES_MARILL},
+    }}, {{ // Spoink
+        {EVO_LEVEL, 32, SPECIES_GRUMPIG},
+    }}, {{ // Grumpig
+    }}, {{ // Plusle
+    }}, {{ // Minun
+    }}, {{ // Mawile
+    }}, {{ // Meditite
+        {EVO_LEVEL, 37, SPECIES_MEDICHAM},
+    }}, {{ // Medicham
+    }}, {{ // Swablu
+        {EVO_LEVEL, 35, SPECIES_ALTARIA},
+    }}, {{ // Altaria
+    }}, {{ // Wynaut
+        {EVO_LEVEL, 15, SPECIES_WOBBUFFET},
+    }}, {{ // Duskull
+        {EVO_LEVEL, 37, SPECIES_DUSCLOPS},
+    }}, {{ // Dusclops
+    }}, {{ // Roselia
+    }}, {{ // Slakoth
+        {EVO_LEVEL, 18, SPECIES_VIGOROTH},
+    }}, {{ // Vigoroth
+        {EVO_LEVEL, 36, SPECIES_SLAKING},
+    }}, {{ // Slaking
+    }}, {{ // Gulpin
+        {EVO_LEVEL, 26, SPECIES_SWALOT},
+    }}, {{ // Swalot
+    }}, {{ // Tropius
+    }}, {{ // Whismur
+        {EVO_LEVEL, 20, SPECIES_LOUDRED},
+    }}, {{ // Loudred
+        {EVO_LEVEL, 40, SPECIES_EXPLOUD},
+    }}, {{ // Exploud
+    }}, {{ // Clamperl
+        {EVO_TRADE_ITEM, ITEM_DEEP_SEA_TOOTH, SPECIES_HUNTAIL},
+        {EVO_TRADE_ITEM, ITEM_DEEP_SEA_SCALE, SPECIES_GOREBYSS},
+    }}, {{ // Huntail
+    }}, {{ // Gorebyss
+    }}, {{ // Absol
+    }}, {{ // Shuppet
+        {EVO_LEVEL, 37, SPECIES_BANETTE},
+    }}, {{ // Banette
+    }}, {{ // Seviper
+    }}, {{ // Zangoose
+    }}, {{ // Relicanth
+    }}, {{ // Aron
+        {EVO_LEVEL, 32, SPECIES_LAIRON},
+    }}, {{ // Lairon
+        {EVO_LEVEL, 42, SPECIES_AGGRON},
+    }}, {{ // Aggron
+    }}, {{ // Castform
+    }}, {{ // Volbeat
+    }}, {{ // Illumise
+    }}, {{ // Lileep
+        {EVO_LEVEL, 40, SPECIES_CRADILY},
+    }}, {{ // Cradily
+    }}, {{ // Anorith
+        {EVO_LEVEL, 40, SPECIES_ARMALDO},
+    }}, {{ // Armaldo
+    }}, {{ // Ralts
+        {EVO_LEVEL, 20, SPECIES_KIRLIA},
+    }}, {{ // Kirlia
+        {EVO_LEVEL, 30, SPECIES_GARDEVOIR},
+    }}, {{ // Gardevoir
+    }}, {{ // Bagon
+        {EVO_LEVEL, 30, SPECIES_SHELGON},
+    }}, {{ // Shelgon
+        {EVO_LEVEL, 50, SPECIES_SALAMENCE},
+    }}, {{ // Salamence
+    }}, {{ // Beldum
+        {EVO_LEVEL, 20, SPECIES_METANG},
+    }}, {{ // Metang
+        {EVO_LEVEL, 45, SPECIES_METAGROSS},
+    }}, {{ // Metagross
+    }}, {{ // Regirock
+    }}, {{ // Regice
+    }}, {{ // Registeel
+    }}, {{ // Kyogre
+    }}, {{ // Groudon
+    }}, {{ // Rayquaza
+    }}, {{ // Latias
+    }}, {{ // Latios
+    }}, {{ // Jirachi
+    }}, {{ // Deoxys
+    }}, {{ // Chimecho
+    }}
```

Comment:
What are the double braces for? At first I thought it was some clever way to avoid `{0}` termination but it's fixed length.